siliconcompiler/scheduler.py

```diff
@@ -111,7 +111,8 @@ def _defernode(chip, step, index):
         with open(script_path, 'w') as sf:
             sf.write('#!/bin/bash\n')
             sf.write(f'sc -cfg {shlex.quote(cfg_file)} -builddir {shlex.quote(buildir)} '
-                     f'-arg_step {shlex.quote(step)} -arg_index {shlex.quote(index)}\n')
+                     f'-arg_step {shlex.quote(step)} -arg_index {shlex.quote(index)} '
+                     f'-from {shlex.quote(step)} -to {shlex.quote(step)}\n')
```

Comment:
why is this needed, if we provide `arg_step` and `arg_index`

Reply:
Sure I can just make it such that `arg_step` overwrites `from` and `to` it's value.